evcouplings/couplings/protocol.py

```diff
@@ -294,63 +295,13 @@ def standard(**kwargs):
 
     # infer ECs and load them
     outcfg, ecs, segments = infer_plmc(**kwargs)
+    model = CouplingsModel(outcfg["model_file"])
 
     # following computations are mostly specific to monomer pipeline
-
-    # add mixture model probability
-    ecs = pairs.add_mixture_probability(ecs)
-
-    # write updated table to csv file
-    ecs.to_csv(outcfg["ec_file"], index=False)
-
-    # also store longrange ECs as convenience output
-    if kwargs["min_sequence_distance"] is not None:
-        outcfg["ec_longrange_file"] = prefix + "_CouplingScores_longrange.csv"
-        ecs_longrange = ecs.query(
-            "abs(i - j) >= {}".format(kwargs["min_sequence_distance"])
-        )
-        ecs_longrange.to_csv(outcfg["ec_longrange_file"], index=False)
-
-        # also create line-drawing script (for now, only for single segments)
-        if segments is None or len(segments) == 1:
-            outcfg["ec_lines_pml_file"] = prefix + "_draw_ec_lines.pml"
-            L = outcfg["num_sites"]
-            ec_lines_pymol_script(
-                ecs_longrange.iloc[:L, :],
-                outcfg["ec_lines_pml_file"]
-            )
-
-    # compute EC enrichment (for now, for single segments
-    # only since enrichment code cannot handle multiple segments)
-    if segments is None or len(segments) == 1:
-        outcfg["enrichment_file"] = prefix + "_enrichment.csv"
-        ecs_enriched = pairs.enrichment(ecs)
-        ecs_enriched.to_csv(outcfg["enrichment_file"], index=False)
-
-        # create corresponding enrichment pymol scripts
-        outcfg["enrichment_pml_files"] = []
-        for sphere_view, pml_suffix in [
-            (True, "_enrichment_spheres.pml"), (False, "_enrichment_sausage.pml")
-        ]:
-            pml_file = prefix + pml_suffix
-            enrichment_pymol_script(ecs_enriched, pml_file, sphere_view=sphere_view)
-            outcfg["enrichment_pml_files"].append(pml_file)
-
-    # output EVzoom JSON file if we have stored model file
-    if outcfg.get("model_file", None) is not None:
-        outcfg["evzoom_file"] = prefix + "_evzoom.json"
-        with open(outcfg["evzoom_file"], "w") as f:
-            # load parameters
-            c = CouplingsModel(outcfg["model_file"])
-
-            # create JSON output and write to file
-            f.write(
-                evzoom_json(c) + "\n"
-            )
-
-    # dump output config to YAML file for debugging/logging
-    write_config_file(prefix + ".couplings_standard.outcfg", outcfg)
-
+    is_single_segment = segments is None or len(segments) == 1
+    _postprocess_inference(ecs, kwargs, model, outcfg, prefix,
```

Comment:
1) please adjust line wrapping like this for consistency with rest of code:
_postprocess_inference(
    ecs, kwargs, model, outcfg, prefix,
    generate_enrichment=is_single_segment,
    generate_line_plot=is_single_segment
)

2) _postprocess_inference shouldn't have side effects besides writing files - especially not on outcfg
I really like the new dictionary unpacking syntax, so in the end this looks like this like in other protocols:
```
outcfg = {
    **outcfg,
   **_postprocess_inference(...),
}
```

Reply:
Made the requested changes.